scripts/build_ckan_dataset_description.py

```diff
@@ -71,7 +71,10 @@ def build_ckan_dataset_description(strOutputFileName, number_of_objects, strFilt
         ckan_dataset_description["url"] = "https://hub.worldpop.org/"
         ckan_dataset_description["license_id"] = "cc-by"
     elif source == "srtm":
-        ckan_dataset_description["url"] = "https://www.earthdata.nasa.gov/"
+        ckan_dataset_description["url"] = "https://www.usgs.gov/"
+        ckan_dataset_description["license_id"] = "other-pd"
+    elif source == "gmted250":
```

Comment:
Is not this source **gmted2010** ?

Reply:
Compare it with line 24 of 
there we have gmted2010
`${outdir}/${country_code}_elev_dem_ras_s0_gmted2010_pp_elevation250m.tif"`

Source is that segment!

Reply:
i fixed the error now